include/vkwrap/vkwrap/mman.h

```diff
@@ -0,0 +1,556 @@
+#pragma once
+
+#include "common/vulkan_include.h"
+#include <vk_mem_alloc.h>
+
+#include "utils/misc.h"
+#include "vkwrap/core.h"
+#include "vkwrap/error.h"
+
+#include <range/v3/algorithm/any_of.hpp>
+
+#include <range/v3/range/conversion.hpp>
+
+#include <array>
+#include <memory>
+#include <unordered_map>
+#include <utility>
+
+namespace std
+{
+
+template <> struct hash<vk::Buffer>
+{
+    size_t operator()( const vk::Buffer& buffer ) const
+    {
+        return reinterpret_cast<size_t>( buffer.operator VkBuffer() );
+    } // operator()
+
+}; // struct hash<vk::Buffer>
+
+template <> struct hash<vk::Image>
+{
+    size_t operator()( const vk::Image& image ) const
+    {
+        return reinterpret_cast<size_t>( image.operator VkImage() );
+    } // operator()
+
+}; // struct hash<vk::Image>
+
+} // namespace std
+
+namespace vkwrap
+{
+
+inline bool
+hasStencil( vk::Format format )
+{
+    using vk::Format;
+
+    constexpr auto k_formats_with_stencil = std::array{
+        Format::eS8Uint,
+        Format::eD16UnormS8Uint,
+        Format::eD24UnormS8Uint,
+        Format::eD32SfloatS8Uint,
+    };
+
+    auto isEqualFormats = [ format ]( Format format_with_stencil ) {
```

Comment:
Naming. Lambda is a variable, so i think it can be named is_equal_formats. On the other hand, this lambda is short and doesn't have to be named. Consider the following: 

```cpp
return ranges::any_of( k_formats_with_stencil, [ format ] ( Format ) { .... } );
```

Reply:
Agree.